docs/learn-more.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
what do you think about linking this doc from the repo's main README.md?

Reply:
good idea, will do